src/app/organisms/emoji-board/EmojiBoard.jsx

```diff
@@ -333,7 +337,7 @@ function EmojiBoard({ onSelect, searchRef }) {
           </ScrollView>
         </div>
         <div ref={emojiInfo} className="emoji-board__content__info">
-          <div>{ parse(twemoji.parse('🙂')) }</div>
+          <div>{ parse(twemoji.parse('🙂', {base: cdn})) }</div>
```

Comment:
we can directly return the base url to options instead of creating module scoped `cdn` variable.